test/services/school/setDefaultStudentListPermission.integration.test.js

```diff
@@ -4,16 +4,16 @@ const appPromise = require('../../../src/app');
 
 const testObjects = require('../helpers/testObjects')(appPromise);
 
-describe('school student list permission scenarios for ADMIN_TOGGLE_STUDENT_VISIBILITY', () => {
-	describe('describe scenario for non-NBC/enabled', () => {
+describe('school student list permission scenarios for TEACHER_STUDENT_VISIBILITY__IS_ENABLED_BY_DEFAULT', () => {
+	describe('scenario for enabled', () => {
```

Comment:
when TEACHER_STUDENT_VISIBILITY__IS_ENABLED is false
--- its
when TEACHER_STUDENT_VISIBILITY__IS_ENABLED is true
--- when TEACHER_STUDENT_VISIBILITY__IS_CONFIGURABLE  is false
--- --- its
--- when TEACHER_STUDENT_VISIBILITY__IS_CONFIGURABLE  is true
--- --- its

Reply:
Nothing was tested here regarding the configurability before. All four tests tested the setting of the default when a school is created. Two of those were in fact duplicates which I removed. 

I wasn't sure if to add new tests for configurability as this is part of the legacy server.